### DISCUSSION TITLE
헤더, 푸터 어떻게 할지

### DISCUSSION THREAD

**Discussion by JunYoung-C** [TEAM-TEMPUS/jjapturi#35] [General]
1. 헤더, 푸터를 담은 jsp를 만들어서 include로 가져오기
2. 각 html 페이지에 복사-붙여넣기하기

**Comment by wyu950** [comment]
로그인, 회원가입, 회원정보 수정에는 html파일 header footer 넣고 다른 페이지들은 헤더, 푸터를 담은 jsp만들어서 include로 가져오기

**Reply by yongki1403** [reply]
jsp로 만들어서 include 가져오는 방법이 좋을 것 같네요

**Comment by lifetruman** [comment]
진행해보다가 불편한 사항이 있으면 html 파일에 header footer 넣는것도 좋은 방법인것 같아요!!

**Reply by JunYoung-C** [reply]
좋은 방법인거 같아요
